# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../jed/fix_utf8.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/misc.c.orig	2006-03-04 09:55:05.000000000 +0200
+++ ./src/misc.c	2006-11-03 21:43:25.000000000 +0200
@@ -741,7 +741,7 @@
 
 void jed_ungetkey_wchar (SLwchar_Type wc)
 {
-   SLuchar_Type *b, buf[SLUTF8_MAX_MBLEN];
+   SLuchar_Type *b, buf[JED_MAX_MULTIBYTE_SIZE];
 
    if (NULL == (b = jed_wchar_to_multibyte (wc, buf)))
      return;
